test/policies/rate-limiter/rate-limit.by.host.test.js

```diff
@@ -25,7 +25,7 @@ describe('rate-limit by host', () => {
                 action: {
                   max: 1,
                   // eslint-disable-next-line no-template-curly-in-string
-                  rateLimitBy: '${req.host}'
+                  rateLimitBy: '${req.hostname}'
```

Comment:
What's going on here?  Does Express add a `req.hostname` but not `req.host`?  Seems weird to have to go the header for the latter.  In any case, might want to either change this to use the Host header or change the name of the file/test from `host` to `hostname`.  These mean two different things.

Reply:
I've checked and they're the same thing.

> express deprecated req.host: Use req.hostname instead

Reply:
but in express 5 it will be back
host is hostname:port

Reply:
Ah! I didn't know that!
I should check this thing more carefully then. I'll write back once I finish to investigate.

Reply:
Maybe they're the same thing in Express's vocabulary, but they're not the same thing to HTTP, so this is confusing.

* hostname: `example.org`
* host: `example.org:80`